RELEASE_PROCESS.md

```diff
@@ -74,73 +104,48 @@ Creating a new .z release
 1. The AWS step functions are now running; proceed with release notes and MacOS
    builds as for `.0` releases; do not update `version.h` in master.
 
-Building A MacOS Release On Azure
```

Comment:
deleted; obsolete. Now orchestrated by the AWS step function (which is mentioned elsewhere in this doc)